advanced/attributes/src/main/java/org/arakhne/afc/attrs/attr/AttributeType.java

```diff
@@ -152,53 +155,118 @@
 		NAME_RESOURCE_FILE = pName + ".types"; //$NON-NLS-1$
 	}
 
+	/** FactoryMap collection.
+	 *
+	 */
+	private static final Map<AttributeType, StringFactory> FACTORY_MAP =
+			Collections.unmodifiableMap(new HashMap<AttributeType, StringFactory>() {{
```

Comment:
Ok, I understand. The first block is for the type definition, and the second inner block is the instance initialisation block. It's ok for me.